.travis.yml

```diff
@@ -20,7 +20,8 @@ before_install:
     - sudo apt-get install -qq ros-indigo-opencv3
     - sudo rosdep init
     - rosdep update
-
+    # install and run pylint
+    #- sudo apt-get install pylint
```

Comment:
What's the command for running pylint? Can you add it in the `script` section of `.travis.yml`?